docs/release-notes.md

```diff
@@ -6,6 +6,20 @@ hide:
 
 # Release Notes
 
+## 0.21.1
+
+### Changed
+
+- from_kwargs doesn't require model or table anymore. It is simply ignored.
```

Comment:
**Breaking**: `from_kwargs` doesn't require model or table anymore. It is simply ignored.`